src/components/TextField/tests/TextField.test.tsx

```diff
@@ -190,6 +190,17 @@ describe('<TextField />', () => {
       );
       expect(textField.find('input').prop('autoComplete')).toBe('on');
     });
+
+    it('sets autoComplete to string value when string is given', () => {
+      const textField = shallowWithAppProvider(
+        <TextField
+          label="TextField"
+          autoComplete="happyString"
```

Comment:
This is a bit of a nitpick, but I wonder if the example in the test case ought to better illustrate a real-world use-case for passing a string. That way folks reading the tests can learn more about its usage. What do you think?

Reply:
👍 will change the string to "fieldName" since thats the intention for it.